godb/.yamllint

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
Do you think we should have one of these per role? I've moved this file to the root of the repo in https://github.com/hic-infra/shared-services-ansible-roles/pull/3

Reply:
I think a shared one is good! I can remove this.